src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElseThrow(() ->
+                new DataProcessingException("Can't get actor with id: " + id));
```

Comment:
```suggestion
                new NoSuchElementException("Actor with id " + id + " wasn't found"));
```
or
```suggestion
                new EntityNotFoundException("Actor with id " + id + " wasn't found"));
```
This exception should not be related to problems with DB, just that we don't have requested data.
Please recheck all services for same